pkgs/tools/admin/awscli/default.nix

```diff
@@ -38,7 +38,7 @@ in with py.pkgs; buildPythonApplication rec {
     bcdoc
     s3transfer
     six
-    colorama
+    colorama043
```

Comment:
I find it highly unlikely that it is incompatible with a different patch release version. Please test patching out the pinning first.

Reply:
Yep, will do, just waiting to have the time.